Cargo.toml

```diff
@@ -10,6 +10,6 @@ documentation = "https://docs.rs/plane-split"
 
 [dependencies]
 binary-space-partition = "0.1.2"
-euclid = "0.11.2"
+euclid = ">= 0.11, < 0.14"
```

Comment:
Because I didn't want it to be a breaking change. Since I need to bump the version to 0.4.0 anyway, i'll bump euclid to 0.13